webapp/views/App/views/Data/Records/Records.js

```diff
@@ -1,26 +1,26 @@
-import './recordsView.scss'
+import './Records.scss'
 
 import React from 'react'
 import { useHistory } from 'react-router'
 
 import * as Record from '@core/record/record'
 
-import { useSurveyCycleKey } from '@webapp/store/survey'
+import { useSurveyCycleKey, useSurveyNodeDefs } from '@webapp/store/survey'
 
 import { appModuleUri, dataModules } from '@webapp/app/appModules'
 
 import Table from '@webapp/components/Table'
 
-import RecordsHeaderLeft from './components/recordsHeaderLeft'
-import RecordsRowHeader from './components/recordsRowHeader'
-import RecordsRow from './components/recordsRow'
+import HeaderLeft from './HeaderLeft'
+import RowHeader from './RowHeader'
+import Row from './Row'
 
-const RecordsView = () => {
+const Records = () => {
   const history = useHistory()
   const cycle = useSurveyCycleKey()
-  const nodeDefKeys = [] // TODO: fetch them
+  const nodeDefKeys = useSurveyNodeDefs()
```

Comment:
add a hook useNodeDefRootKeys that returns an array of nodeDefs that are keys of the root entity

Reply:
if i understand well this are the keys of the root entity:

<img width="656" alt="Captura de pantalla 2020-06-17 a las 9 47 01" src="https://user-images.githubusercontent.com/11272068/84870422-9aa41080-b07f-11ea-8f65-5c62fe75b7fc.png">

Reply:
Nope. By keys it’s meant the attribute that are marked as key